pkg/awsManager/efsManager.go

```diff
@@ -72,7 +84,7 @@ func DeleteEFSMountTarget(client clientpkg.Client, mountTargetToBeDeleted []*str
 		localMetrics.ResourceSuccess(localMetrics.EfsVolume, client.GetRegion())
 	}
 
-	if mountTargetNotDeleted != nil {
+	if len(mountTargetNotDeleted) > 0 {
```

Comment:
I don't think this needed to change, was it causing problems?

Reply:
Nope I don't think it was, I just tend to fallback to length checking arrays out of habit